packages/contracts/tasks/deploy-teleportr.ts

```diff
@@ -0,0 +1,40 @@
+/* Imports: External */
+import { ethers } from 'ethers'
+import { task } from 'hardhat/config'
+import * as types from 'hardhat/internal/core/params/argumentTypes'
```

Comment:
Perhaps out of scope for this PR, but I do think it would also be helpful to have tasks for teleportr admin stuff like changing the config